prep-javascript-variables/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Sara Roohan';
+
+    var numberOfPets = 0;
+
+    var likesVegetables = true;
+
+    var nothing = null;
+
+    var never;
```

Comment:
Oop! Looks like your console logs didn't get included

Reply:
Oh, yes, I actually took those out since the reminder at the bottom of the logging-to-the-console page said to remove this type of before turning it in, but now that I'm re-reading it, obviously this assignment was the exception!! I'll add that back in now. Thanks!

Reply:
Updated!
![image](https://user-images.githubusercontent.com/78943858/109112087-eb7c0680-76ee-11eb-943a-bbf7061c21e1.png)